sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/DiskEncryptionSetUpdate.cs

```diff
@@ -41,12 +41,16 @@ public DiskEncryptionSetUpdate()
         /// <param name="rotationToLatestKeyVersionEnabled">Set this flag to
         /// true to enable auto-updating of this disk encryption set to the
         /// latest key version.</param>
+        /// <param name="federatedClientId">Multi-tenant application client id
+        /// to access key vault in a different tenant. Setting the value to
+        /// 'None' will clear the property.</param>
         /// <param name="tags">Resource tags</param>
-        public DiskEncryptionSetUpdate(string encryptionType = default(string), KeyForDiskEncryptionSet activeKey = default(KeyForDiskEncryptionSet), bool? rotationToLatestKeyVersionEnabled = default(bool?), IDictionary<string, string> tags = default(IDictionary<string, string>), EncryptionSetIdentity identity = default(EncryptionSetIdentity))
+        public DiskEncryptionSetUpdate(string encryptionType = default(string), KeyForDiskEncryptionSet activeKey = default(KeyForDiskEncryptionSet), bool? rotationToLatestKeyVersionEnabled = default(bool?), string federatedClientId = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), EncryptionSetIdentity identity = default(EncryptionSetIdentity))
```

Comment:
Although the major version is bumped, it would still be better to provide backward compatibility customizations like: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations

Reply:
please see my comments below, I've added the customizations.